test/e2e/specs/recordedit/helpers.js

```diff
@@ -360,13 +360,13 @@ exports.testPresentationAndBasicValidation = function(tableParams) {
 	                            var rows;
 	                            chaisePage.clickButton(popupBtns[(columns.length * recordIndex) + i ]).then(function() {
 	                                // wait for the modal to open
-	                                browser.wait(EC.visibilityOf(modalTitle), 5000);
+	                                browser.wait(EC.visibilityOf(modalTitle), browser.params.defaultTimeout);
 
 	                                return modalTitle.getText();
 	                            }).then(function(text) {
 	                                // make sure modal opened
 	                                expect(text.indexOf("Choose")).toBeGreaterThan(-1);
-
+                                    // TODO: What is this 5-sec sleep for? To make sure all rows are loaded?
```

Comment:
I believe that is correct. This is so to make sure the rows all load first. This should certainly be turned into a `browser.wait()`.